LoM-Install/arista/do-install.py

```diff
@@ -0,0 +1,768 @@
+    """
+    logger.log("Starting lom-engine daemon ...")
+    try:
+        # Start the lom-engine daemon in management namespace
+        g_switch_eapi_handler.add_lom_engine_daemon(g_lom_engine_path, g_config_dir, RUN_MODE, 6, True)
+        logger.log("lom-engine started successfully")
+    except Exception as e:
+        logger.log("Error while starting lom-engine: {}".format(e))
+        raise e
+    
+    # Wait for some time after starting the daemon
+    time.sleep(DAEMON_AFTER_ENABLED_WAIT_TIME)
+
+    # Check if lom-engine daemon is running
+    try:
+        running = g_switch_eapi_handler.is_daemon_running('lom-engine')
+    except Exception as e:
+        logger.log("Error while checking if lom-engine is running: {}".format(e))
+        raise e
+
+    if not running:
+        logger.log("lom-engine is not running")
+        return False
+
+    logger.log("lom-engine is running successfully")
+    return True
+    #To-Do : Goutham : call is_process_port_in_namespace() to check
+    
+def check_and_start_plugin_manager_deamon():
+    """
+    This function is responsible for starting the lom-plugin-manager daemon for each proc_id and checking if it's running.
+
+    It performs the following steps:
+    1. Iterate over each proc_id in g_procs_keys
+    2. Start the lom-plugin-manager daemon for each proc_id
+    3. Wait for some time after starting the daemon
+    4. Check if the lom-plugin-manager daemon is running for each proc_id
+
+    Returns:
+    bool: True if all lom-plugin-manager daemons are running, False otherwise.
+    """
+    # Iterate over each proc_id in g_procs_keys and start lom-plmgr for each proc_id            
+    for proc_id in g_procs_keys:
+        # Start the lom-plugin-manager daemon
+        logger.log("Starting lom-plmgr for proc_id {} ...".format(proc_id))
+        try:
+            g_switch_eapi_handler.add_plugin_manager_daemon(g_lom_plugin_mgr_path, proc_id, g_config_dir, RUN_MODE, 6, True)
+            logger.log("lom-plmgr started successfully for proc_id {}".format(proc_id))            
+        except Exception as e:
+            logger.log("Error while starting lom-plmgr for proc_id {}: {}".format(proc_id, e))
+            raise e
+        
+        time.sleep(DAEMON_AFTER_ENABLED_WAIT_TIME)
+        instance_name = "lom-plmgr-{}".format(proc_id)
+
+        # Check if lom-plmgr daemon is running
+        try:
+            running = g_switch_eapi_handler.is_daemon_running(instance_name)
+        except Exception as e:
+            logger.log("Error while checking if {} is running: {}".format(instance_name, e))
+            raise e
+
+        if not running:
+            logger.log("{} is not running".format(instance_name))
+            return False
+        logger.log("{} is running successfully".format(instance_name))            
+    return True
+    #To-Do : Goutham : call is_process_port_in_namespace() to check
+        
+def pre_checks():
```

Comment:
add To-Do comment : Add precheck to match eos version with the supported lom installer version.

Reply:
added